src/www/ui/core-schema.dat

```diff
@@ -1484,6 +1484,10 @@
   $Schema["TABLE"]["upload"]["public_perm"]["ADD"] = "ALTER TABLE \"upload\" ADD COLUMN \"public_perm\" int4";
   $Schema["TABLE"]["upload"]["public_perm"]["ALTER"] = "ALTER TABLE \"upload\" ALTER COLUMN \"public_perm\" DROP NOT NULL";
 
+  $Schema["TABLE"]["upload"]["scm"]["DESC"] = "COMMENT ON COLUMN \"upload\".\"scm\" IS 'true: automatically ignore SCM data when scaning, false: don\'t ignore it'";
```

Comment:
Can you please fix the `don\'t`, it is failing in the Travis tests.
https://travis-ci.org/fossology/fossology/jobs/475316319#L2821

Reply:
Also please try to rebase your branch with fossology master as there are some changes showing in your branch which are already merged by #1232

Reply:
Thanks. OK I will update core-shema.dat
```suggestion
  $Schema["TABLE"]["upload"]["scm"]["DESC"] = "COMMENT ON COLUMN \"upload\".\"scm\" IS 'true: automatically ignore SCM data when scaning, false: dont ignore it'";
```
I don’t understand what changes done by me are already merged by #1232.

Reply:
One of the changes: (Your PR is on the left and #1232 is on the right)

![2019-01-07 17_09_35-virtualbox](https://user-images.githubusercontent.com/18077542/50766417-24eb2100-129f-11e9-8b64-fd263f9eb49e.png)

Reply:
OK .travis.yml is now updated with fossology master version.

Reply:
Hello @ChristopheRequillart, your PR still does not look in good shape. From the commits, I guess what you did was a merge of fossology:master to your branch.

Kindly rebase your branch to fossology:master. For help, please follow the [contributing guideline](https://github.com/fossology/fossology/blob/master/CONTRIBUTING.md#pull-requests).

Reply:
Hello @GMishx. I do a rebase as write in [contributing guideline](https://github.com/fossology/fossology/blob/master/CONTRIBUTING.md#pull-requests). But I have to do a merge to resolve diverging branches.